ledger/complete/mtrie/trie/trie.go

```diff
@@ -94,9 +94,9 @@ func (mt *MTrie) RootNode() *node.Node {
 // Concurrency safe (as Tries are immutable structures by convention)
 func (mt *MTrie) String() string {
 	if mt.IsEmpty() {
-		return fmt.Sprintf("Empty Trie with default root hash: %x\n", mt.RootHash())
+		return fmt.Sprintf("Empty Trie with default root hash: %v\n", mt.RootHash())
```

Comment:
`RootHash()` returns `hash.Hash`, which has implemented `String` method already, using `%x` would double encoding.

For instance,
Before the fix, it prints:
```
Empty Trie with default root hash: 35363866346563373430666533623564653838303334636237623166626464623431353438623036386633316165626338616539313839653432396335373439
```

After the fix, it prints:
```
Empty Trie with default root hash: 568f4ec740fe3b5de88034cb7b1fbddb41548b068f31aebc8ae9189e429c5749
```